src/game/Spell.cpp

```diff
@@ -6244,6 +6265,12 @@ bool Spell::CheckTarget(Unit* target, SpellEffectIndex eff)
     // A player can cast spells on his pet (or other controlled unit) though in any state
     if (target != m_caster && target->GetCharmerOrOwnerGuid() != m_caster->GetObjectGuid())
     {
+        switch (m_spellInfo->Id) // spells that target minipets, which are inherently non attackable
+        {
+            case 44877:
```

Comment:
I do not follow, I am targeting minipet(critter), how would adding spell_script_target help if minipets cant be targeted by anything